DOMA-4288 send tin and user info with user message

VKislov opened the pull request:
sending user info with user messages in useDesk for support staff

sitozzz commented on apps/condo/domains/common/components/UseDeskWidget.tsx:
```diff
@@ -1,9 +1,33 @@
+                additional_fields:
+                    [
+                        {
+                            id: 20560, value: get(link, ['organization', 'tin'], null),
```
May be move id to constants or config?

sitozzz commented on apps/condo/domains/common/components/UseDeskWidget.tsx:
```diff
@@ -1,9 +1,33 @@
 const UseDeskWidget: React.FC = () => {
+    const { link } = useOrganization()
+
+    const userIdentify = useCallback(() => {
```
I think useCallback here is unnecessary

VKislov commented:
Removed `JivoSiteWidget` from the code. Initially, it was planned to disable it through the `JivoSite` admin panel, but they have a bug in the admin panel and so far it is impossible to disable the widget remotely. Therefore, in order to avoid overlapping widgets, I removed it from the code. I remember about the config in Helm, I'll drink it in one of the following
